src/pudl/transform/ferc1.py

```diff
@@ -2138,20 +2155,58 @@ def apply_xbrl_calculation_fixes(
                 ],
             },
             "balance_sheet_assets_ferc1": {
+                "current_and_accrued_assets": [
+                    {
+                        "calc_component_to_replace": {
+                            "name": "noncurrent_portion_of_allowances",
+                            "weight": 1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                        "calc_component_new": {
+                            "name": "less_noncurrent_portion_of_allowances",
+                            "weight": -1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                    },
+                    {
+                        "calc_component_to_replace": {
+                            "name": "derivative_instrument_assets_long_term",
+                            "weight": 1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                        "calc_component_new": {
+                            "name": "less_derivative_instrument_assets_long_term",
+                            "weight": -1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                    },
+                    {
+                        "calc_component_to_replace": {
+                            "name": "derivative_instrument_assets_hedges_long_term",
+                            "weight": 1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                        "calc_component_new": {
+                            "name": "less_derivative_instrument_assets_hedges_long_term",
+                            "weight": -1.0,
+                            "source_tables": ["balance_sheet_assets_ferc1"],
+                        },
+                    },
+                ],
                 "nuclear_fuel_net": [
                     {
                         "calc_component_to_replace": {
                             "name": "nuclear_fuel_materials_and_assemblies",
                             "weight": 1.0,
-                            "source_tables": ["balance_sheet_assets_ferc1"],
+                            "source_tables": [],
```

Comment:
I think these might show up in another dedicated nuclear fuel table actually.

And I think there may be some naming confusing where in one table they end with `_major_only` and in another they don't?